### PR TITLE
ROADMAP: add 2019's roadmap

### PR DESCRIPTION
**Pull request by SantiagoTorres**
**Fixes issue #**: None

**Description of the changes being introduced by the pull request**:
This adds the year-long roadmap for the in-toto python reference implementation

**Please verify and check that the pull request fulfills the following
requirements**:

- N/A The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- N/A Tests have been added for the bug fix or new feature
- [x] Docs have been added for the bug fix or new feature

### CODE REVIEW

**Comment by JustinCappos** [comment]
Looks great!  I assume we're being a little light on technical and integration details intentionally.  Is the thinking that this is better left underspecified here.

**Comment by SantiagoTorres** [comment]
Yes, I wanted to keep things a little bit open so we could adjust the specifics as times go by.

I intend to use the roadmap review windows to figure out how things are going and on how to clarify how these goals are going to be met.

**Comment by reza-curtmola** [comment]
Looks good!

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+This document spans the Roadmap for the in-toto main implementation for the
+time window from April 2019 to April 2020. The main theme of this year's
+efforts are focused on two fronts strengthening our release process and aiming
+for version 1.0!
+
+## Strengthening the release process
+
+As a software supply chain security product, in-toto aims to be at the
+forefront of the best practices when it comes to developing the tool. As such,
+the in-toto has the following goals for this year
```
Comment (lukpueh) [inline ROADMAP.md:13]: either remove the *"the"* or add something after *"in-toto"* (e.g. team, project, organization, ...)

Comment (lukpueh) [reply ROADMAP.md:13]: Does this need a period or colon at the end of the sentence?

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+### CII Silver Badge
+
+in-toto is already marked as "passing criteria" under the Core Infrastructure
+Initiative best practices badge. This year we aim to upgrade our rating to
+Silver. To work towards this goal, we'll review and comply with the few
+missing requirements for the silver badge.
+
+### Thorough Documentation of Release process with in-toto metadata
```
Comment (lukpueh) [inline ROADMAP.md:22]: Is the mix of capitalization and lower-case intended? The rest of the document has only the first word of the section headings capitalized (except for CII Silver Badge).

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+in-toto is integrated in different products already, which is a strong
+motivation to hit the 1.0 milestone. This year we intend to reach our 1.0
+release.
+
+### The 1.0 milestone
+
+A GitHub project will be announced with the tasks required for the 1.0
+milestone by the in-toto team leaders. This milestone will center towards the
+reaching 1) a stable API for the library and the CLI 2) a thorough
```
Comment (lukpueh) [inline ROADMAP.md:40]: reaching *of* 1) a stable ...

File: ROADMAP.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+## Release scheduling and roadmap review schedule
+
+As part of the 1.0 efforts, we intend to follow a more thorough release
+schedule. This includes a more formal Service Level Agreement considering a
+four month window between minor versions.
+
+- Months 1-2: Normal development, including new features, refactoring, and bugfixes.
+- Month 3: Feature freeze: only bugfixes and security vulnerabilities are allowed.
+- Month 4: Release candidate period. A new version is released by the end of
+  the fourth month.
```
Comment (lukpueh) [inline ROADMAP.md:75]: Okay, I admit, this is super nit-picking, but do you mind settling for one punctuation mark after the phases? Currently you have:

Normal development, (coma)
Feature freeze: (colon)
Release candidate period. (period)

It's okay if you do mind. :)

**Review by lukpueh — COMMENTED** [review]
Awesome write-up, @SantiagoTorres! Below just a few proof-reading-style comments.

**Comment by lukpueh** [comment]
@SantiagoTorres, let me know if you want me to send a PR to address https://github.com/in-toto/in-toto/pull/267#pullrequestreview-222742606.

**Comment by SantiagoTorres** [comment]
@lukpueh sounds good to me, please make the PR.

**Pull request merged**